### PR TITLE
feat: Adds output-file and fail-on-result

### PR DESCRIPTION
**Pull request by rafaeldelboni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,19 @@ Simply add the following snippet to your GitHub actions workflow.
   uses: clj-holmes/clj-holmes-action@main
 ```
 
-It's also possible to specify a custom rule-repository (public or private) and the output-type (sarif or stdout)
+## Inputs
+It's also possible to specify the following:
+- rule-repository, it can be an public or private git repository [Default: git://clj-holmes/clj-holmes-rules#main]
+- output-type (sarif or stdout) [Default: sarif]
```
Comment (mthbernardes) [inline README.md]: there is also a json output type

File: entrypoint.sh [hunk 2]
```diff
@@ -2,7 +2,14 @@
...
+if [[ $fail_on_result == "true" ]]; then
+  fail_on_result_cmd="--fail-on-result";
+else
+  fail_on_result_cmd="--no-fail-on-result";
+fi
 
 cd /github/workspace/
 clj-holmes fetch-rules -r "$rules_repository"
-clj-holmes scan -p . -t "$output_type" -o clj-holmes.sarif
+clj-holmes scan -p . $fail_on_result_cmd -t "$output_type" -o clj-holmes.sarif
```
Comment (mthbernardes) [inline entrypoint.sh]: since now it supports specify the output type what do u think about adding an option to specify the filename?

Comment (mthbernardes) [reply entrypoint.sh]: Also if the output-type is stdout it does not need the output-file option

**Review by mthbernardes — COMMENTED** [review]
:rocket:

File: entrypoint.sh (hunk 2 above)
Comment (rafaeldelboni) [reply entrypoint.sh]: :+1:

File: README.md (hunk 1 above)
Comment (rafaeldelboni) [reply README.md]: :+1:

**Review by mthbernardes — APPROVED** [review]
:rocket: :no_entry: :back:

**Pull request merged**